tests/visual-snapshots.spec.ts

```diff
@@ -60,4 +61,7 @@ const takeFullPageScreenshot = (component, storyName) => {
   return ScreenshotOptions[component]?.fullPageScreenshot || ScreenshotOptions[storyName]?.fullPageScreenshot;
 };
 
-fs.writeFileSync(`./tests/snapshots/used-screenshot-paths-${matrixKey}.txt`, usedScreenshotPaths.join('\n'));
+fs.writeFileSync(
+  path.join(`.`, `tests`, `snapshots`, `used-screenshot-paths-${matrixKey}.txt`),
+  usedScreenshotPaths.join('\n')
+);
```

Comment:
```suggestion
constunusedScreenshotsFilePath = path.join(`.`, `tests`, `snapshots`, `used-screenshot-paths-${matrixKey}.txt`);
fs.writeFileSync(unusedScreenshotsFilePath,usedScreenshotPaths.join('\n'));
```

extract this one to a variable like the stories file path?

Reply:
corrected